setup.cfg

```diff
@@ -0,0 +1,30 @@
+[coverage:report]
```

Comment:
Do you think it's worth adding package metadata and options like [this](https://github.com/pystatgen/sgkit-plink/blob/master/setup.cfg#L1) now?

Reply:
Done in 14e15146e562dfe8c0e632807c08a8c63ef9f458